app/views/DistributionsIncludedView.scala.html

```diff
@@ -69,12 +69,12 @@ <h1 class="govuk-fieldset__legend--l">
                 items = Seq(
                     RadioItem(
                         content = Text(messages("distributionsIncluded.option1Label")),
-                        value = Some(messages("distributionsIncluded.option1Value")),
+                        value = Some(messages("site.yes")),
                         conditionalHtml = Some(yesHtml)
                     ),
                     RadioItem(
                         content = Text(messages("distributionsIncluded.option2Label")),
-                        value = Some(messages("distributionsIncluded.option2Value"))
+                        value = Some(messages("site.No"))
```

Comment:
site.No will that work if its upper case?

Reply:
Also, the values is constant across languages. You should be using site.yes and site.not for content, not the value. Value should be hardcoded, regardless of the language
 value = Some("yes")

value = Some("no")